docs/SDK/Explanation/migration.md

```diff
@@ -5,218 +5,211 @@ sidebar_position: 2
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Migrating to V2
+# Migrating From V2 To V3
 
-`lit-js-sdk` is now considered deprecated and will only receive security updates. It is strongly recommended to migrate to the new **Lit JS SDK V2** for continued support and new features.
+The **Lit JS SDK V3** replaces the existing access control condition based encryption and JWT signing processes with new cryptographic primitives to offer a more secure and seamless user experience.
 
-The **Lit JS SDK V2** has been entirely revamped from the earlier `lit-js-sdk` and completely rewritten in TypeScript. This latest V2 SDK is much more modular and easier to use, and has a noticeably smaller bundle size.
+Refer to the [changelog](https://github.com/LIT-Protocol/js-sdk/blob/master/CHANGELOG.md) for more details.
```

Comment:
the rest of this page seems to go over the changes, so the reference to the changelog might not be necessary as the changelog tends to be brief and only points to the commit/pr.